planner/core/explain.go

```diff
@@ -696,8 +699,33 @@ func (p *TiKVSingleGather) ExplainInfo() string {
 
 // ExplainInfo implements Plan interface.
 func (p *PhysicalMemTable) ExplainInfo() string {
-	if !metricschema.IsMetricTable(p.Table.Name.L) {
+	return genMetricTableExplainInfo(p.ctx, p.DBName, p.Table, p.Extractor)
+}
+
+// ExplainInfo implements Plan interface.
+func (p *LogicalMemTable) ExplainInfo() string {
+	return genMetricTableExplainInfo(p.ctx, p.dbName, p.tableInfo, p.Extractor)
+}
+
+func genMetricTableExplainInfo(ctx sessionctx.Context, dbName model.CIStr, tblInfo *model.TableInfo, extractor MemTablePredicateExtractor) string {
+	if dbName.L != util.MetricSchemaName.L || !metricschema.IsMetricTable(tblInfo.Name.L) {
+		return ""
+	}
+
+	e, ok := extractor.(*MetricTableExtractor)
+	if !ok || e.SkipRequest {
+		return ""
+	}
+	quantile, err := e.GetQuantile()
+	if err != nil {
```

Comment:
The error is just ignored? You can append a warning to the `StatementContext` at least.

Reply:
ok, but normally, it can't be an error.